deploy-manage/tools.md

```diff
@@ -5,28 +5,48 @@ mapped_pages:
 
```

Comment:
please add the `applies` tag with the relevant products inside of that front matter block 

https://elastic.github.io/docs-builder/syntax/applies.html